src/peergos/server/tests/MultiUserTests.java

```diff
@@ -493,6 +495,37 @@ public void followPeergos() throws Exception {
         assertTrue("Receive a follow request", u1Requests.size() > 0);
     }
 
+    @Test
+    public void sendFeedbackToPeergos() throws Exception {
+        UserContext peergos = PeergosNetworkUtils.ensureSignedUp("peergos", "testpassword", network, crypto);
+        UserContext newUser = PeergosNetworkUtils.ensureSignedUp(random(), "newUserPassword", network, crypto);
+
+        // Check that new user can send feedback to the user peergos.
+        String feedback = "Here's some constructive feedback!";
+        CompletableFuture<Boolean> testFeedbackSubmission = newUser.submitFeedback(feedback);
+        assertTrue("Feedback submission was successful!", testFeedbackSubmission.get() == true);
+
+        List<FollowRequestWithCipherText> peergosRequests = peergos.processFollowRequests().get();
+        peergos.sendReplyFollowRequest(peergosRequests.get(0), true, true).get();
```

Comment:
after this you could call newUser.processFollowRequests() which finalises the bidirectional following, though it shouldn't be needed.